helm/prometheus-rules/templates/kaas/tenet/alerting-rules/certificate.all.rules.yml

```
@@ -25,7 +25,7 @@ spec:
     - alert: KyvernoCertificateSecretWillExpireInLessThanTwoDays
       annotations:
         description: '{{`Kyverno Certificate stored in Secret {{ $labels.exported_namespace }}/{{ $labels.name }} on {{ $labels.cluster_id }} will expire in less than two days.`}}'
-        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/kyverno-certificate-secret-will-expire-in-less-than-two-days/
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/kyverno-certificate-secret-will-expire-in-less-than-two-days/?cluster_id={{ $labels.cluster_id }}
```

I think it requires the whole value to be evaluated, just like for `description`:
```suggestion
        runbook_url: '{{`https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/kyverno-certificate-secret-will-expire-in-less-than-two-days/?cluster_id={{ $labels.cluster_id }}`}}'
```